### ISSUE TITLE
Cannot load spells & spell slots on Arcane Trickster

### ISSUE THREAD

**Issue by xdaft** [stinodes/combat#38]
Everything else seems to load without any problems.

**Comment by stinodes** [comment]
Yeah, some (sub)-classes seem to be a bit iffy in getting spellcasting to load.
Cantrips also arent displayed. Woops.
